src/static/examples/loadScripts.js

```diff
@@ -0,0 +1,27 @@
+var fluid_load = fluid_load || {};
```

Comment:
I don't think there is any implication here. The code is under our definition and I don't see any reason to take up a separate top-level namespace - there's nothing about the definition of the "fluid" namespace that implies how complete its contents should be. In fact in future we will positively promote the use of "fluid bootstrappers" which enable low-powered clients get access to framework facilities cheaply without necessarily loading, for example, all of Fluid IoC.